torch/_torch_docs.py

```diff
@@ -4338,6 +4338,48 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(self, other, *, out=None) -> Tensor
+
+Computes ``self * log(other)`` except it treats ``0 * log(0)`` as ``0``.
+
+Similar to SciPy's `scipy.special.xlogy`.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
+        \text{NaN} & \text{if } \text{other}_{i} = \text{NaN} \\
+        0 & \text{if } \text{self}_{i} = 0.0 \\
+        self_i * \log{(other_i)} & \text{otherwise}
+    \end{cases}
+
+""" + r"""
+
+Args:
+    self (Number or Tensor)
+    other (Number or Tensor)
+
+.. note:: Both attr:`self` and attr:`other` cannot be number.
```

Comment:
"Both attr:`self` and attr:`other` cannot be number." -> "At least one of :attr:\`self\` or :attr:\`other\` must be a tensor."